meta/main.yml

```diff
@@ -8,11 +8,14 @@ galaxy_info:
   platforms:
     - name: Amazon
       versions:
-        - 2018.03
+        - 2
```

Comment:
How does this work?  It's not in the list: https://aws.amazon.com/amazon-linux-ami/

Reply:
These are Docker images, not AMIs.  See [here](https://hub.docker.com/_/amazonlinux/?tab=description) for more details.